man/ukify.xml

```diff
@@ -112,6 +112,22 @@
 
       <para>The output files must not exist.</para>
     </refsect2>
+
+    <refsect2>
+      <title><command>inspect</command></title>
+
+      <para>Display information about the given PE, listing all known sections and
+      print the content of each (if text readable) together with displaying its size and
+      <literal>sha256</literal>.
+      This mode has already a list of default sections to show, but it is possible to specify
```

Comment:
"By default only the most useful sections will be shown. It is possible to override the list of sections to show, by specifying …".